src/main/features/offer/views/offer-sent-confirmation.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton %}
+{% set title = t("Money claim") %}
+{% set heading = t("Your offer has been sent to the claimant") %}
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+        <p>
+        {{ t("We'll email the claimant %s confirming the offer.",
```

Comment:
may need to be smarter here because claimant and defendant's can make offers too each other.
Can leave to improve later though..